src/test/java/tech/jhipster/lite/module/infrastructure/secondary/JHipsterModulesAssertions.java

```diff
@@ -82,6 +82,14 @@ public static ModuleFile vitestConfigFile() {
     return fileFromClasspath("generator/typescript/vitest.config.ts.mustache", "vitest.config.ts");
   }
 
+  public static ModuleFile viteConfigFile() {
+    return fileFromClasspath("generator/typescript/vite.config.ts.mustache", "vite.config.ts");
```

Comment:
Since we don't have an unique source template in main resource for vite.config (react has its own template, and so do vue), this file should not be put in **main** resources, but rather in **test** resources:
```suggestion
    return fileFromClasspath("projects/typescript-with-vite/vite.config.ts.mustache", "vite.config.ts");
```